website/pages/docs/platform/k8s/helm/configuration.mdx

```diff
@@ -25,6 +25,8 @@ and consider if they're appropriate for your deployment.
 
   - `tlsDisable` (`boolean: true`) - When set to `true`, changes URLs from `https` to `http` (such as the `VAULT_ADDR=http://127.0.0.1:8200` environment variable set on the Vault pods).
 
+  - `openshift` (`boolean: false`) - If `true`, enables configuration specific to OpenShift such as NetworkPolicy, SecurityContext, and Route.
```

Comment:
Can you add a note about this being a beta feature?

Reply:
Done in 31d7de9